README.md

```diff
@@ -1 +1,27 @@
 # Istio 101
+
+This is the repository for the [Istio][istio] 101 workshop from IBM.
+
+- [presentation](./presentation) -- the data for the teacher/leader of the workshop
+- [workshop](./workshop) -- the vanilla code for the code of the workshop
+
+
+```text
+Copyright:: IBM, Inc 2018-
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Do we need this section since we have a LICENSE file?  At a minimum I would point people to the local file and not the remote one

Reply:
The theory is you can just look at the README and see the license without clicking through.

There are companies/people that before touching an opensource repo have to identify the License.